styles.css

```diff
@@ -0,0 +1,78 @@
+html {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: sans-serif;
+  margin: 0;
+}
+
+/* .container {
+  display: flex;
+  flex-direction: column;
+} */
```

Comment:
- [ ] Consider removing these commented lines of code as it makes our code messy. Let's keep our code clean by removing them. What do you think? 🤔

Reply:
Hi @franciscoPonceDev  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉🎉🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._